crates/aggchain-proof-contracts/src/config.rs

```diff
@@ -1,4 +1,4 @@
-use alloy::primitives::{address, Address};
+use agglayer_primitives::{address, Address};
```

Comment:
I think it makes sense for `agglayer-proof-contracts` to have `agglayer_primitives::Address` in the APIs, as it will be used across the agglayer components, despite the overhead of conversions.